### ISSUE TITLE
100% CPU usage opening bevyengine.org

### ISSUE THREAD

**Issue by ggiraldez** [brave/brave-browser#29568]
## Description 

Opening the URL https://bevyengine.org with Brave Version 1.49.120 Chromium: 111.0.5563.64 (Official Build) (64-bit) makes the main brave process CPU usage pin at 100% (as reported by `top`).

Closing the tab has no effect and the process still uses 100% CPU. Opening a second (and third, fourth and so forth) increases the CPU usage by 100% each time, up to ~800% (my computer has 8 cores). After opening the same site multiple times, the CPU usage seems to stabilize around 600%.

Quitting the browser does not kill the process. The only way I found so far to get rid of it is to `kill -9` it.

## Steps to Reproduce

   1. Open https://bevyengine.org
   2. Close the tab
   3. Go to step 1

## Actual result:

CPU usage increases by 100% for each new tab opened, up to the maximum allowed by the number of cores.

## Expected result:

Opening a tab does not eat my CPU usage, or at least closing the tab returns the usage to normal parameters.

## Reproduces how often:

Always.

## Brave version (brave://version info)

1.49.120 Chromium: 111.0.5563.64 (Official Build) (64-bit) on Ubuntu Linux 20.04.5 LTS installed by apt using the official repository at https://brave-browser-apt-release.s3.brave.com/

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by rebron** [comment]
@ggiraldez Can you update to our latest 1.50.114 and let us know if you're still seeing the issue.

**Comment by ggiraldez** [comment]
> @ggiraldez Can you update to our latest 1.50.114 and let us know if you're still seeing the issue.

Yep, just updated and the problem persists. BUT, opening new tabs no longer increase the usage above 100%. I closed all the tabs and the usage is still at 100%.

**Comment by HeroTch** [comment]
Same on Windows, Brave Browser v1.50.114
